pkg/oc/admin/prune/images.go

```diff
@@ -58,9 +58,9 @@ var (
 		Insecure connection is allowed in the following cases unless certificate-authority is
 		specified:
 
-		 1. --force-insecure is given  
-		 2. provided registry-url is prefixed with http://  
-		 3. registry url is a private or link-local address  
+		 1. --force-insecure is given
```

Comment:
This change breaks formatting. The spaces were there to preserve line breaks. Without them, the lines will be concatenated.